h_transport_materials/property_database/palladium.py

```diff
@@ -472,6 +472,14 @@
     source="powell_surface_1991",
 )
 
+takagi_recombination_h = RecombinationCoeff(
+    pre_exp=1.5e-27 * 6.022e23 * u.m**4 * u.s**-1 * u.mol**-1,  #convert to mols
+    act_energy=0.48 * 6.022e23 * u.eV * u.mol**-1, #convert to mols
+    range=(280 * u.K, 568 * u.K),
+    isotope="H",
```

Comment:
This study is with Deuterium

```suggestion
    isotope="D",
```